src/batch/client.ts

```diff
@@ -114,7 +115,7 @@ export class BatchTranscription {
     format = 'json-v2',
   }: TranscribeConfig): Promise<RetrieveTranscriptResponse | string> {
     if (this.config.apiKey === undefined)
-      throw new Error('Error: apiKey is undefined');
+      throw new SpeechmaticsInternalError('Error: apiKey is undefined');
```

Comment:
I'll revisit this so that we pass in the enum to uses of SpeechmaticsInternalError.